src/client.js

```diff
@@ -1287,36 +1281,29 @@ function SugarApi(args) {
          * @param {Object} [$files] jQuery/Zepto DOM elements that carry the files to upload.
          * @param {Object} [callbacks] callback object.
          * @param {Object} [options] Request options hash.
+         * @param {boolean} [options.iframe=true] If `true`, use iframe transport.
```

Comment:
OOPS! I don't think this option even has any effect any more. Make sure to check that before sending to CR.